bitbots_convenience_frames/src/convenience_frames.cpp

```diff
@@ -0,0 +1,209 @@
+#include <ros/ros.h>
+#include <std_msgs/Char.h>
+#include <tf2_ros/transform_broadcaster.h>
+#include <tf2_ros/transform_listener.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2/utils.h>
+#include <humanoid_league_msgs/BallRelative.h>
+#include <humanoid_league_msgs/GoalRelative.h>
+#include <humanoid_league_msgs/GoalPartsRelative.h>
+
+#include <utility>
+
+class ConvenienceFramesBroadcaster {
+ public:
+  ConvenienceFramesBroadcaster();
+ private:
+  tf2_ros::TransformBroadcaster broadcaster_;
```

Comment:
Now only `broadcaster_` seem to be default initialize. Anything else looks good to me.

Reply:
done